crates/miden-objects/src/account/code/mod.rs

```diff
@@ -285,6 +285,36 @@ impl Deserializable for AccountCode {
     }
 }
 
+// PRETTY PRINT
+// ================================================================================================
+
+impl PrettyPrint for AccountCode {
+    fn render(&self) -> vm_core::prettier::Document {
+        use vm_core::prettier::*;
+        let mut partial = Document::Empty;
+        let len_procedures = self.procedures().len();
+        for (index, procedure_root) in self.procedure_roots().enumerate() {
+            let node_id = self
+                .mast
+                .find_procedure_root(procedure_root)
+                .expect("procedure root should be present in the mast forest");
+            let node_raw = self.mast[node_id].clone();
+
+            partial = partial
+                + indent(
+                    4,
+                    const_text("begin") + nl() + node_raw.to_pretty_print(&self.mast).render(),
+                )
+                + nl()
```

Comment:
Not sure what's the best format, but it may be good to also print out storage info for each procedure (i.e., offset and size).

Also, I'm not too sure about using `being` and `end` as a way to delineate procedures. Maybe it should look something like:
```
export."<procedure_digest>"
    <procedure body>
end
```

Reply:
This is how the latest implementation is being printed, it has the offset and size included, and uses a `PrintableProcedure` struct for each procedure like the mentioned in the other comment:

<img width="1030" alt="image" src="https://github.com/user-attachments/assets/d6c63c7f-5c65-4d33-aba2-87a55b822d24" />

Reply:
I don't know how to properly print the procedure body just using the `AccountCode`.